README.md

```diff
@@ -23,11 +23,11 @@ By the way, creating issues is fun too!
 ![Screenshot](gif/jump.gif)
 
 **NB!**
- - **Your pull-request may not be reviewed very quickly, please have patience.**
- - **If you need help with a PR (e.g. resolving merge conflict) ask the community for support**
-   **posting a message `#help_wanted` + link to your pull request**
-   **( e.g. https://github.com/kottans/mock-repo/pull/145 )**
-   **in a chat of the course you're taking.**
+- **Your pull-request may not be reviewed very quickly, please have patience.**
+- **If you need help with a PR (e.g. resolving merge conflict) ask the community for support**
+  **posting a message `#help_wanted` + link to your pull request**
+  **( e.g. https://github.com/kottans/mock-repo/pull/145 )**
+  **in a chat of the course you're taking.**
```

Comment:
Don't change text of other people

Reply:
But the linter shows a warning, and the "README" say:
> The change can be anything you like, for example: fixing a typo in README.md.

[md007---unordered-list-indentation](https://github.com/DavidAnson/markdownlint/blob/v0.21.1/doc/Rules.md#md007---unordered-list-indentation)